packages/fleet/fleet/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/fleet/releases/download/v0.8.1/fleet-0.8.1.tgz
-version: 102.2.1
+url: https://github.com/rancher/fleet/releases/download/v0.8.2-rc.3/fleet-0.8.2-rc.3.tgz
```

Comment:
This means that we would be providing a release candidate version of fleet in an official helm chart once we merge `dev-v2.7.11` to `release-v2.7`. Seems fishy to me. Why does QA need us to package a helm chart that contains an RC? Shouldn't we only be packaging full releases?

Reply:
I hope this is temporary, Fleet team seems to have updated the Dockerfile on Rancher to point to this version but they merged this version on dev-v2.7 branch only.